### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by hirsch88** [baloise-incubator/design-system#212]
Each component should be testet with the cypress a11y plugin.

Example is the bal-tag.spec file

 

We test the basic component
We test the disable state
We test the color variants
We test active and inactive state
Fix a11y issues if possible
 

 

All our components should meet the double AA standard

Accessibility Conformity
WCAG
Size WCAG 2.1 AA: Link
Text Spacing WCAG 2.1 AA: Link
Contrast WCAG 2.1 AA: Link
https://www.w3.org/TR/WCAG21/#info-and-relationships (A)
https://www.w3.org/TR/WCAG21/#keyboard (A)
https://www.w3.org/TR/WCAG21/#label-in-name (A)
https://www.w3.org/TR/WCAG21/#non-text-contrast (AA)
https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus (AA)
https://www.w3.org/TR/WCAG21/#target-size (optional as AAA)

Components:

- [ ] Accordion
- [ ] Badge
- [ ] Button
- [ ] Card
- [ ] Close
- [ ] Data
- [ ] Heading
- [ ] Hint
- [ ] Icon
- [ ] Image Slider
- [ ] List
- [ ] Logo
- [ ] Navbar
- [ ] (Navigation)
- [ ] Pagination
- [ ] Product Slider
- [ ] Shape
- [ ] Stage
- [ ] Table
- [ ] Tabs
- [ ] Tabs Steps
- [ ] Tag
- [ ] Text
- [ ] Checkbox
- [ ] Radio
- [ ] Datepicker
- [ ] Field
- [ ] FielUpload
- [ ] Input Stepper
- [ ] Input
- [ ] Number Input
- [ ] Select
- [ ] Slider
- [ ] Textarea
- [ ] Time Input
- [ ] Notification
- [ ] Toast
- [ ] Snackbar
- [ ] Modal
- [ ] Sheet
